app/actions/remote/entry/gql_common.ts

```diff
@@ -303,6 +303,10 @@ export const graphQLCommon = async (serverUrl: string, syncDatabase: boolean, cu
         await operator.batchRecords(models.flat());
     }
 
+    if (isNotificationEntry) {
+        await handleNotificationNavigation(serverUrl, initialChannelId, initialTeamId, currentChannelId, currentTeamId, rootId);
```

Comment:
This seems to be called a bit late

Reply:
Why? As before, it is called after the initial batching, and before the deferred actions.

Reply:
that is why I said "seems" :P